doc/chapter-podcasts.asciidoc

```diff
@@ -1,13 +1,20 @@
 A podcast is a media file distributed over the internet using syndication feeds
-such as RSS, for later playback on portable players or computers.  Newsboat
-contains support for downloading and saving podcasts. This support differs a bit
-from other podcast aggregators or "podcatchers" in how it is done.
+such as RSS, for later playback on portable players or computers.
+Newsboat supports downloading, saving and streaming podcasts, though an external
+media player is needed for playback. This support differs a bit from other
+podcast aggregators or "podcatchers" in how it is done.
 
 Podcast content is transported in RSS feeds via special tags called
 "enclosures". Newsboat recognizes these enclosures and stores the relevant
 information for every podcast item it finds in an RSS feed. Since version 2.0,
-it also recognizes and handles the Yahoo Media RSS extensions. What the user
-then can do is to add the podcast download URL to a download queue.
+it also recognizes and handles the Yahoo Media RSS extensions.
+
+The following sections present two different ways to handle podcasts with
+newsboat.
+
+==== Managing Audio Files with Podboat
```

Comment:
Thanks for the tip, I will try and build the docs soon enough.

It's because "Podcast Support" chapter is in its own file `chapter-podcasts.asciidoc`, so I didn't know how to write the heading.

I'd say we should either move "Open Links with External Commands" to it's own file too, or bring back "Podcast Support" to `newsboat.asciidoc`.

Any preference?

Reply:
Ah, shoot: I was inclined to move everything to _newsboat.asciidoc_, so checked where we use the _doc/chapter-podcasts.asciidoc_ file, and turns out we also include it in the Podboat's manpage. The chapter now talks about Newsboat macros, which looks completely out of place in Podboat's manpage.

I feel like the best solution would be to replace the chapter with a few paragraphs explaining the purpose of Podboat and referring users to the HTML doc for details. That way, we keep the manpage on-topic, while also preserving a nice outline that you've created in the HTML doc. Do you have any alternative proposals, @A1RO?

Reply:
Well, that's a bummer. It explains the separate chapter though. There are a few things in there to think through.
Allow me some time to build the app and doc in the next few days. I'd like to try local commits to test the end result.

Reply:
Okay, looking forward to see what you come up with ^_^

Reply:
I just built the whole thing. I can see what you mean when looking at `man podboat`. I also see the updated html version for the first time at `/usr/share/doc/newsboat/newsboat.html`. Better late than never!

Now that I have the right tools, I'm sure I can come up with something.
I might be able to push some commits tomorrow.

Reply:
Cool! Note that you don't have to install Newsboat in order to check on the docs, it's sufficient to do this:

```
$ make doc
$ man doc/newsboat.1
$ firefox doc/xhtml/newsboat.html
```

Reply:
Have a look at today's commits.

_newsboat.html_ looks identical for the user, but podboat's manpage looks pretty clean now.
I didn't write a single extra word of documentation.